hs_core/forms.py

```diff
@@ -980,8 +1133,8 @@ def clean(self):
 
 
 def get_crispy_form_fields(field_names, file_type=False):
-    """
-    This creates a list of objects of type Field
+    """Return a list of objects of type Field.
+
```

Comment:
@aphelionz I see you have used this commenting format consistently, so it is your call which format works better and you can ignore my comment above.